qiskit/circuit/library/generalized_gates/mcmt.py

```diff
@@ -282,6 +282,7 @@ def _ccx_v_chain_rule(
         Raises:
             QiskitError: If an insufficient number of ancilla qubits was provided.
         """
+        # TODO: should it actually return anyting?
```

Comment:
Here, the type annotation claims to return some list, but the function doesn't return anything